lib/xclarity_client/error/connection_failed_unknown.rb

```diff
@@ -0,0 +1,10 @@
+module XClarityClient
+  module Error
+    class ConnectionFailedUnknown < StandardError
+      def initialize(msg)
```

Comment:
Should we set a default value of _nil_ for _msg_ like you do in the other error classes?

Reply:
Hi Stephen, firstly thank you for reviewing. So, I thought about forcing the passage of a message when this kind of exception occurs, because an error of any kind might happen, like 404 or 302 and son on.